pkg/apiserver/registry/datapackaging/package_storage_client.go

```diff
@@ -152,7 +153,30 @@ func (t PackageTranslator) ToExternalWatcher(intObjWatcher watch.Interface, fiel
 }
 
 func (t PackageTranslator) ToExternalError(err error) error {
-	// TODO: implement
+
+	status, ok := err.(apierrors.APIStatus)
+	if !(ok || errors.As(err, &status)) {
+		return err
+	}
```

Comment:
Should this code be something like this?
```suggestion
	var status *apierrors.APIStatus
	if !(errors.As(err, &status)) {
		return err
	}
```

Reply:
This was a copy from https://github.com/kubernetes/apimachinery/blob/12dc3f82eb47fc972b5be0b222f4422cb1b51612/pkg/api/errors/errors.go#L99. I am okay with both the approaches.